packages/documentation-framework/templates/mdx.js

```diff
@@ -126,6 +143,9 @@ export const MDXTemplate = ({
   id,
   componentsData
 }) => {
+  const isDeprecated = sources.some(source => source.source === "react-deprecated") && !sources.some(source => source.source === "react");
+  const isBeta = sources.some(source => source.beta)
+  const isDemo = sources.some(source => source.source === "react-demos" || source.source === "html-demos") && !sources.some(source => source.source === "react" || source.source === "html");
```

Comment:
Something we're guilty of throughout docs-framework are these PF-specific hard coded values.  Now that the docs-framework is a package used outside just the PF website build we should try to allow these to be customized by the user whenever possible.

Reply:
FYI this is widespread throughout the package, this works for now but noting for possible future cleanup effort.

Reply:
yeah and i'm not sure if I can programmatically determine that a deprecated label can be added to the Example title without some sort of data being manually stored in the component-data. That was going to be my fall back, but this felt like less maintenance for now.